### PR TITLE
pytorch multitask models

### PR DESCRIPTION
**Pull request by miaecle**
This PR adds in:

- Implementation of vanilla multitask model in pytorch

Structure is mostly the same as the tensorflow version multitask model, still working on a few improvements: model saving and loading / porting calculations to  GPU
#479

### CODE REVIEW

**Comment by rbharath** [comment]
@miaecle Great to see this! This is awesome, and I'm looking forward to using this code as a foundation to build on for our PyTorch port. Here are a couple comments on what we'll need before merging in:

- An update to `README.md` with installation directions for PyTorch
- A test added to `test_overfit.py` adding correctness tests.
- (Optional but would be very cool) Adding a separate Pytorch performance section to `README.md`. We should measure both accuracy, and runtime (it would be great to benchmark PyTorch vs. Tensorflow for molecular machine learning problems). Maybe just Tox21 numbers for this first PR.

**Comment by miaecle** [comment]
@rbharath Yeah, thanks for the quick response! There are still quite a lot TODOs for the model, planning to work on it in next week.
Also, suggestions or requests are much welcomed and appreciated!

**Comment by rbharath** [comment]
@miaecle Following our discussion offline and #489, the PyTorch code might find a natural home in the `Contrib` folder. Once this code matures, we can move it into main DeepChem for downstream users.

**Comment by miaecle** [comment]
@rbharath Done! Models support GPU calculation now. Two example codes added in the `example/` folder, do we need to set up unit test for models?

**Comment by miaecle** [comment]
@rbharath Still failing due to yapf, is it possible to change yapf settings to exclude `contrib` folder

**Comment by rbharath** [comment]
@miaecle Awesome, thanks for getting this working! Agreed we should exclude `Contrib/` from `yapf`.

**Comment by kexul** [comment]
Hi, how about the performance of this pytorch implement? I test it and it got train mean roc-auc score of 0.98 but 0.64 in validation.I used the tox21_torch.py.
Any help is appreciated! @miaecle
